honeybadgermpc/hbavss.py

```diff
@@ -334,28 +424,30 @@ def _get_dealer_msg(self, values):
         logger.debug("[%d] Starting Batch AVSS. Id: %s, Dealer Id: %d, Client Mode: %s",
                      self.my_id, avss_id, dealer_id, client_mode)
 
+        broadcast_msg = None
         dispersal_msg_list = None
         if self.my_id == dealer_id:
-            dispersal_msg_list = self._get_dealer_msg(values)
+            # broadcast_msg: phi & public key for reliable broadcast
+            # dispersal_msg_list: the list of payload z
+            broadcast_msg, dispersal_msg_list = self._get_dealer_msg(values)
 
         # In the client_mode, the dealer is the last node
         n = self.n if not client_mode else self.n+1
 
         tag = f"{dealer_id}-{avss_id}-B-AVID"
         send, recv = self.get_send(tag), self.subscribe_recv(tag)
 
+        rbc_msg = await reliablebroadcast(
```

Comment:
add a log statement before and after, this helps in debugging what got stuck where